pillar.example

```diff
@@ -94,12 +96,17 @@ netweaver:
       instance: 00
       root_user: root
       root_password: linux
+      # Set the shared disk used in HA environments. Skip this parameter in non HA environments
       shared_disk_dev: /dev/sbd
+      # Init the shared disk. Only used if a shared disk is provided, not in nfs share cases
       init_shared_disk: True
-      sap_instance: ascs
+      # If a nfs share is used to manage the HA mounting point, like in the cloud
```

Comment:
maybe this comment and `shared_disk_dev` could be moved after `shared_disk_dev: /dev/sbd ` and start with `Or if a nfs share...`